crates/examples/src/readobj/pe.rs

```diff
@@ -309,6 +310,61 @@ fn print_import_dir<Pe: ImageNtHeaders>(
     Some(())
 }
 
+fn print_delay_load_dir<Pe: ImageNtHeaders>(
+    p: &mut Printer<'_>,
+    data: &[u8],
+    sections: &SectionTable,
+    data_directories: &DataDirectories,
+) -> Option<()> {
+    let import_table = data_directories
+        .delay_load_import_table(data, sections)
+        .print_err(p)??;
+    let mut import_descs = import_table.descriptors().print_err(p)?;
+    p.group("ImageDelayLoadDirectory", |p| {
+        while let Some(Some(import_desc)) = import_descs.next().print_err(p) {
+            p.group("ImageDelayLoadDescriptor", |p| {
+                let dll_name = import_desc.dll_name_rva.get(LE);
+                p.field_string("DllName", dll_name, import_table.name(dll_name));
+                p.field_hex("ModuleHandle", import_desc.module_handle_rva.get(LE));
+                p.field_hex(
+                    "ImportAddressTable",
+                    import_desc.import_address_table_rva.get(LE),
+                );
+                p.field_hex("ImportNameTable", import_desc.import_name_table_rva.get(LE));
+                p.field_hex(
+                    "BoundImportAddressTable",
+                    import_desc.bound_import_address_table_rva.get(LE),
+                );
+                p.field_hex(
+                    "UnloadInformationTable",
+                    import_desc.unload_information_table_rva.get(LE),
+                );
+                p.field_hex("TimeDateStamp", import_desc.time_date_stamp.get(LE));
+
+                let mut name_thunks = import_table
+                    .thunks(import_desc.import_name_table_rva.get(LE))
+                    .print_err(p);
```

Comment:
So this is displaying the INT. Should we display the IAT too, similar to how we do in `print_import_dir`? Also, would it be useful to display the BIAT and UIAT?

Reply:
I think there may be some values in the .data that we could display, but they are low interest so this is fine for now.